controllers/vpcendpoint/helpers.go

```diff
@@ -167,6 +227,12 @@ func (r *VpcEndpointReconciler) findOrCreateVpcEndpoint(resource *avov1alpha1.Vp
 		vpce = resp.VpcEndpoints[0]
 	}
 
+	resource.Status.VPCEndpointId = *vpce.VpcEndpointId
```

Comment:
Add a nil check for `vpce` above this deref?

Reply:
Added! Same for when I similarly dereferenced the security group to get its ID.